NewForce-Cohort-6/tabloidfullstack-mashed-potatoes
# 32UnsubscribeToAUsersPosts

# Description
[Trello Ticket #32](https://trello.com/c/mBV2ieDn)

Added functionality to unsubscribe to another user's posts. When viewing a post that a user has subscribed to, an Unsubscribe button will appear. When you click the unsubscribe button, the end time of the subscription gets added to the database and the page reverts to showing a subscribe button. 


# Testing Instructions
1. `git add` and `git commit` whatever you're working on on your branch
2. `git checkout main`
3. `git fetch --a`
4. `git checkout 32UnsubscribeToAUsersPosts`
5. If you don't see the most recent version of my files, `git pull origin 32UnsubscribeToAUsersPosts`
6. If you haven't created the Tabloid database, please do so.
Use this data to fill out tables and columns: [SQL Tables](https://github.com/NewForce-Cohort-6/tabloidfullstack-mashed-potatoes/blob/main/SQL/01_Tabloid_Create_DB.sql)
Use this to seed the database : [SQL SEED DATA](https://github.com/NewForce-Cohort-6/tabloidfullstack-mashed-potatoes/blob/main/SQL/02_Tabloid_Seed_Data.sql)
7. Click the execute with debugger solid green button to launch the app
8. If you are not already logged in to the site, log in.
9. From the home page, click the "Posts" option in the navbar and select a post by another user.
10. You should be redirected to that post's details page and there should be a subscribe button or an unsubscribe button at the top
11. Click subscribe, then unsubscribe, in either order depending on the post you've clicked on
12. The button should be replaced with an unsubscribe button and a note indicating that you're subscribed, or a subscribed button if you've just unsubscribed
13. Try again with another user if you'd like
14. Go back to the post list and select a post by the user you've logged in with. There should not be a button or a note about subscription status on this page

# Checklist:
